packages/klevu-ui/src/components/klevu-dropdown/klevu-dropdown.css

```diff
@@ -7,13 +7,16 @@ select {
 
   font-size: var(--klevu-body-s-size);
   font-weight: var(--klevu-body-s-weight);
-  appearance: none;
+  -webkit-appearance: none;
```

Comment:
prefixes should not be added to code.

Reply:
If they are required we are adding them automatically with processors.

Reply:
removed automatically generated prefixes